### ISSUE TITLE
Incorrect video recording orientation on iPad

### ISSUE THREAD

**Issue by peagasilva** [AvdLee/RocketSimApp#452]
**Describe the bug**
When recording a video in landscape on an iPad, the final orientation is not correct.

**To Reproduce**
1. Go to an iPad simulator
2. Click on 'Record'
3. Stop recording
4. See that the video is upside down

**Expected behavior**
The final video should be with the correct orientation.

**Screenshots**

| Simulator | Result |
:---: | :---:
| <img width=400 src="https://github.com/AvdLee/RocketSimApp/assets/5387088/f874b919-4635-4af1-b866-ebd6412c3479" /> | <video width=200 src="https://github.com/AvdLee/RocketSimApp/assets/5387088/d67fe90a-6298-42b1-8122-c0a27fcb8a3d" /> |

**Desktop (please complete the following information):**
 - OS: Sonoma
 - Version: 14.2.1 (23C71)

**Comment by AvdLee** [comment]
Duplicate of #90 

This is a known issue and something I can't fix. I filled feedback:

```
Run the following command from the terminal:

1. Open an iPad Simulator
2. Turn the iPad into landscape mode
3. Run `xcrun simctl io booted screenshot screenshot.png` in the terminal
4. Notice the screenshot is in portrait while it should be landscape
```

## Anyone reading this
If you run into this issue as well, please duplicate `FB13550695` and let Apple know you're waiting for a fix.

@peagasilva I'll keep this issue open an pin it to the repository.

**Comment by AvdLee** [comment]
Somehow, I overlooked the Simulator _is_ returning the proper orientation right now. I'm going to look into fixing this again!

**Comment by AvdLee** [comment]
Just found this inside the user defaults

```
"FDBA62E7-CA64-4B55-A8E1-9F19988EA323" =         {
            ChromeTint = "";
            SimulatorExternalDisplay = 2114;
            SimulatorWindowGeometry =             {
                "0A03D92B-FBEF-4596-A865-FA28E2647118" =                 {
                    WindowCenter = "{945.5, 581.5}";
                    WindowScale = "0.6679447852760736";
                };
            };
            SimulatorWindowOrientation = LandscapeLeft;
            SimulatorWindowRotationAngle = 90;
        };
```

Might be able to solve this after all!

**Comment by AvdLee** [comment]
So, I'm not able to read these user defaults, but I would be able to ask for permissions when a user tries to record in landscape mode 👌

**Comment by AvdLee** [comment]
This will be fixed in 12.11.0
